backend/src/parser/TheParser.ts

```diff
@@ -221,6 +221,7 @@ export default class TheParser {
 
         // noinspection HtmlDeprecatedAttribute
         return `<iframe width="500" height="282" src="${encodeURI(embed)}" allowfullscreen frameborder="0"></iframe>`;
+
```

Comment:
let's remove this newline so that this file stays untouched

Reply:
done